docs/conf.py

```diff
@@ -37,12 +37,15 @@
 except ImportError:
     import importlib_metadata
 
+from sphinx.errors import NoUri
```

Comment:
Is this import actually used?

Reply:
Nope!  Remnant of an older attempt.  I wonder why flake8 didn't catch it.

Reply:
`flake8` ignores it right now, see #7824